src/FlightDisplay/FlyViewWidgetLayer.qml

```diff
@@ -151,7 +151,38 @@ Item {
         id:                 telemetryPanel
         x:                  recalcXPosition()
         anchors.margins:    _toolsMargin
-        anchors.bottom:     parent.bottom
+
+        // States for custom layout support
+        states: [
+            State {
+                name: "bottom"
+                when: telemetryPanel.bottomMode
+
+                AnchorChanges {
+                    target: telemetryPanel
+                    anchors.top: undefined
+                    anchors.bottom: parent.bottom
+                    anchors.right: undefined
+                }
+
+                PropertyChanges {
+                    target: telemetryPanel
+                    x: recalcXPosition()
+                }
+            },
+
+            State {
+                name: "right"
+                when: !telemetryPanel.bottomMode
+
+                AnchorChanges {
+                    target: telemetryPanel
+                    anchors.top: photoVideoControl.bottom
```

Comment:
The photoVideoControl is not always visible. It can be turned off by a setting.

Reply:
Done, check the latest version of the video.